docker/server/nginx/etc/nginx/conf.d/universe.marathon.mesos.conf

```diff
@@ -48,15 +54,44 @@ server {
 
   location = /repo {
     types {
-      "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v3" json;
       application/zip zip;
     }
 
-    if ($serve_json = true) {
-      rewrite ^/repo$ /repo-up-to-$dcos_release_version.json break;
+    if ($universe_version = v3) {
+      rewrite ^/repo$ /v3;
+    }
+
+    if ($universe_version = v4) {
+      rewrite ^/repo$ /v4;
     }
 
     rewrite ^/repo$ /repo-up-to-$dcos_release_version.zip break;
   }
 
+  location = /v3 {
```

Comment:
This looks like its defining an actual endpoint. Will clients be able to access `https://universe.marathon.mesos/v3`? Is there a way to prevent nginx from routing direct requests to this location?